src/templates/preview/Error.tsx

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import NextHead from 'next/head';
+
+import { ErrorType, IRenderError } from '../../utils/error';
+import { Footer } from '../homepage/Footer';
+
+import { Title } from './Title';
+import { Link, ExternalLink } from '../../components/Link';
+import { QuickLinks } from '../error/QuickLinks';
+
+export * from '../error/ErrorBoundary';
+
+export function Error(error: IRenderError): JSX.Element {
+  return (
+    <>
+      <NextHead>
+        <meta key="noindex" name="robots" content="noindex" />
+        <link rel="preconnect" href="https://fonts.gstatic.com" />
+        {/* eslint-disable-next-line @next/next/no-page-custom-font */}
+        <link href="https://fonts.googleapis.com/css2?family=Anton&display=swap" rel="stylesheet" />
+
+        <link rel="icon" type="image/png" sizes="32x32" href="/favicons/favicon-32x32.png" />
+        <link rel="icon" type="image/png" sizes="16x16" href="/favicons/favicon-16x16.png" />
+      </NextHead>
+      <section className="mt-20 max-w-4xl mx-auto px-2">
+        <Title statusCode={error.statusCode} />
+        <div className="my-16 prose dark:prose-dark max-w-none">
+          {error.statusCode === 500 && <ServerError {...error} />}
+          {error.statusCode !== 500 && <NotFound {...error} />}
+        </div>
+        <QuickLinks />
+        <Footer />
+      </section>
+    </>
+  );
+}
+
+export function ServerError({ properties }: IRenderError): JSX.Element {
+  return (
+    <>
+      <p>Something went wrong whilst building your preview.</p>
+      {!!properties && (
+        <p>
+          The could have happened because of an issue with your local Markdown content, or something
+          internal. If you think that it is not a problem with your markdown, feel free to{' '}
+          <ExternalLink href="https://github.com/invertase/docs.page/issues">
+            report an issue
+          </ExternalLink>
+          .
+        </p>
+      )}
+    </>
+  );
+}
+
+export function NotFound({ properties, errorType }: IRenderError): JSX.Element {
```

Comment:
This file is basically ctrl-c ctrl-v from `/templates/error` and is wip, for now i just changed the 500 message and removed a broken link. Generally error handling is wip still